problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,8 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+function getCountriesSortedByPopulation(arr) {
+    return arr.sort();
```

Comment:
Sort is the right method to think of. By default sort sorts things in alphabetical order. Because we have an array of objects though we have to pass a function to sort to tell it how we want it to sort things. Check the documentation on MDN for sort to see how to sort objects. Lastly we'll need to map through the sorted array to only get the keys we want.